app_dart/lib/src/service/config.dart

```diff
@@ -269,7 +269,13 @@ class Config {
       'resolve the images on the Flutter Gold dashboard, consider marking this PR '
       'as a draft pull request above. You will still be able to view image results '
       'on the dashboard, commenting will be silenced, and the check will not try to resolve itself until '
-      'marked ready for review.\n\n';
+      'marked ready for review.\n\n'
```

Comment:
This will only notify PRs that have been detected as generating untriaged images, it would not have caught today's scenario. We need a new action in the push_gold_status* bot that will notify on any PR that is generating golden file images.

Reply:
Gotcha, PTAL - I think this should be applied now anytime that a comment otherwise would be left.